diff --git a/src/SKSE/RegistrationMap.cpp b/src/SKSE/RegistrationMap.cpp
index f8292a06..815dd8f4 100644
--- a/src/SKSE/RegistrationMap.cpp
+++ b/src/SKSE/RegistrationMap.cpp
@@ -218,7 +218,7 @@ namespace SKSE
 				} else if (keyIndex == 2) {
 					RE::FormType formType = std::get<2>(key);
 					if (!a_intfc->WriteRecordData(formType)) {
-						log::error("Failed to save reg key as formtype ({})!", formType);
+						log::error("Failed to save reg key as formtype ({})!", static_cast<std::underlying_type_t<decltype(formType)>>(formType));
 						return false;
 					}
 				}
